.github/workflows/automerge-import-cwc-repo.yml

```
@@ -0,0 +1,29 @@
+name: automerge-import-cwc-repo
+on:
+  push:
+    branches:
+      - 'main'
+
+concurrency:
+  group: automerge-import-cwc-repo-${{ github.ref }}
+  cancel-in-progress: true
+
+jobs:
+  automerge-mastheadv2:
```

LGTM just one small thing
```suggestion
  automerge-import-cwc-repo:
```

---

gah thanks!